src/core/main.c

```diff
@@ -293,6 +293,36 @@ static int parse_crash_chvt(const char *value) {
         return 0;
 }
 
+static int parse_confirm_spawn(const char *value, char **console) {
+        char *cons;
+        int r;
+
+        r = parse_boolean(value);
+        if (r == 0) {
+                *console = NULL;
+                return 0;
+        }
+        if (r > 0)
+                value = "/dev/console";
+
+        if (strcmp(value, "/dev/console")) {
+                _cleanup_close_ int fd = -1;
+
+                fd = open_terminal(value, O_RDWR|O_NOCTTY|O_CLOEXEC);
+                if (fd < 0) {
+                        log_warning_errno(fd, "failed to open %s: %m, using default console", value);
+                        value = "/dev/console";
+                }
+        }
+
+        cons = strdup(value);
+        if (!cons)
+                return -errno;
+
+        *console = cons;
+        return 0;
+}
```

Comment:
hmm, so ttys are ultimately hw devices. They can come and go any time, and frequently they aren't available yet during early boot (consider a USB rs232 dongle...), hence I think if such a fallback is desirable it should really be done eacht time when asking the question, and not already when we parse the argument here. That way the question stuff will automatically make use of the right tty as soon as it is around, but use /dev/console if it isn't...